components/modal/subscriptions-modal/subscriptions-preview/table/styles.scss

```diff
@@ -0,0 +1,9 @@
+.c-subscription-preview-table {
+  .disclaimer-container {
+    width: 100%;
+    background-color: rgba(250,183,46, 0.1);
```

Comment:
Is this a new color? Don't we have a similar? Sorry for being paranoid with the colors but you know how it can scale if we have no control over this...

Reply:
I actually searched for it before adding it explicitly but couldn't find any occurrences in the code. We could anyway create a CSS variable just in case it's needed in the future. _I'll do that in the next commit._